MAINTAINERS.yml

```diff
@@ -178,12 +175,7 @@ Bluetooth:
         - include/bluetooth/
         - include/drivers/bluetooth/
         - samples/bluetooth/
-        - subsys/bluetooth/*
-        - subsys/bluetooth/common/
-        - subsys/bluetooth/host/
-        - subsys/bluetooth/services/
-        - subsys/bluetooth/shell/
-        - subsys/bluetooth/audio/
+        - subsys/bluetooth/
```

Comment:
This is needed to separate between host and controller, which have 2 different maintainers.

Reply:
Fixed.